tests/configfiles/DockerfileLintBearTest.py

```diff
@@ -24,6 +26,68 @@
 """
 
 
+good_file_2 = """
+FROM ubuntu
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+
+RUN apt-get -y -qq update
+RUN apt-get -y -qq upgrade
+RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+EXPOSE 5432
+CMD ["/usr/bin/node", "/var/www/app.js"]
+"""
+
+
+bad_file_2 = """
+FROM ubuntu
+
+apt-get -y -qq update
+apt-get -y -qq upgrade
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+"""
+
+good_file_3 = """
+FROM kali
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+EXPOSE 5432
+"""
+
+
+bad_file_3 = """
+FROM kali
+LABEL Name
+LABEL Version
+"""
+
+
+def get_testfile_name(filename):
+    return os.path.join(os.path.dirname(__file__),
+                        'dockerfile_lint_test_files',
+                        filename)
+
+
+default_rule = get_testfile_name('default_rule.yaml')
+
+sample_rule = get_testfile_name('sample_rule.yaml')
+
 DockerfileLintBearTest = verify_local_bear(DockerfileLintBear,
                                            valid_files=(good_file,),
                                            invalid_files=(bad_file,))
+
+DockerfileLintBearTest2 = verify_local_bear(
```

Comment:
Could you name your tests a bit so it reflects the fact that you want to test rule files?
Same below.